docs/vt-extensions/color-palette-update-notifications.md

```diff
@@ -43,3 +43,13 @@ e.g. by configuration.
 
 Please have a look at our example C++ [source code](https://github.com/contour-terminal/contour/blob/master/examples/detect-dark-light-mode.cpp)
 in order to see how to implement this in your own application.
+
+## Adoption State
+| Support  | Terminal/Toolkit/App | Notes                                                                   |
+|----------|----------------------|-------------------------------------------------------------------------|
+| ✅       | Contour              |                                                                         |
+| ✅       | Ghostty              | since `1.0.0`                                                           |
+| ✅       | Kitty                | since `0.36.3`                                                          |
```

Comment:
Thank you very much, @bash. However, for Kitty I could not find the relevant PR / release notes adding it. Did just my eyes skip it?

Reply:
I'm assuming it's this commit: https://github.com/kovidgoyal/kitty/commit/dd9d8353dfbb8da655d93fe449ce31be0f0956b6

Reply:
Ah, lovely response times, @j4james. Thank you very much :)

Reply:
Oh, I must have copied the wrong version 😆 It's actually `0.38.1`:

From [kitty's changelog](https://sw.kovidgoyal.net/kitty/changelog/#id1):
> * Add support for [escape code protocol](https://github.com/contour-terminal/contour/blob/master/docs/vt-extensions/color-palette-update-notifications.md) for notifying applications on dark/light color scheme change